tools/ci/scripts/buildNativeProjectLocal.ps1

```diff
@@ -83,14 +83,22 @@ Write-Host "    x64 Build Succeeded:           $64Result"
 Write-Host "    ARM Build Succeeded:           $ARMResult"
 Write-Host "    Copy Native Plugins Succeeded: $CopyResult"
 
-Write-Host "`nIncluded Compositor Components:"
-Write-Host "    Blackmagic Decklink:            " (Test-Path "$PSScriptRoot\..\..\..\external\Blackmagic DeckLink SDK 10.9.11")
-Write-Host "    Elgato:                         " (Test-Path "$PSScriptRoot\..\..\..\external\gamecapture")
-Write-Host "    Azure Kinect SDK:               " (Test-Path "$PSScriptRoot\..\..\..\external\Azure Kinect SDK v1.3.0")
-Write-Host "    Azure Kinect Body Tracking SDK: " (Test-Path "$PSScriptRoot\..\..\..\external\Azure Kinect Body Tracking SDK 1.0.0")
+$DependenciesPropsFile = "$PSScriptRoot\..\..\..\src\SpectatorView.Native\SpectatorView.Compositor\dependencies.props"
+
+if (Test-Path -Path $DependenciesPropsFile -PathType Leaf) {
+  [xml]$DependenciesPropsContent = Get-Content $DependenciesPropsFile
+
+  $SolutionDirVariable = "`$(SolutionDir)"
+  $SolutionDir = "$PSScriptRoot\..\..\..\src\SpectatorView.Native\"
+
+  Write-Host "`nIncluded Compositor Components:"
+  Write-Host "    Blackmagic Decklink:            " (Test-Path $DependenciesPropsContent.Project.PropertyGroup.DeckLink_inc.replace($SolutionDirVariable, $SolutionDir))
```

Comment:
>(Test-Path $DependenciesPropsContent.Project.PropertyGroup.DeckLink_inc.replace($SolutionDirVariable, $SolutionDir)) [](start = 52, length = 116)

If the $(SolutionDir) is no longer in Dependencies.props, do you still need to do the replace here?

Reply:
This step doesn't introduce any additional errors if $(SolutionDir) isn't specified. I'd like to keep this abilityin case paths do get specified relative to the $(SolutionDir) if developers change these dependencies.props locations in their local repo.